tests/Tests/Unit/Common/Auth/OpenIDConnect/Grant/CustomClientCredentialsGrantTest.php

```diff
@@ -0,0 +1,281 @@
+<?php
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/106369572-7a238080-6307-11eb-8846-92414e248be3.gif)

Reply:
I thought you'd like to see how the PHPUnit mock's work, if you've never used them before.  It makes it so you can avoid HTTP requests in unit tests so that test cases run much faster.  I didn't see them anywhere in the codebase so I put some in, one because it helped with the test, and 2 to illustrate by example how to use them.